sdks/python/apache_beam/transforms/trigger.py

```diff
@@ -211,6 +211,16 @@ def should_fire(self, time_domain, timestamp, window, context):
     """
     pass
 
+  @abstractmethod
+  def has_ontime_pane(self):
+    """Whether this trigger creates an empty pane if there are no elements.
```

Comment:
This isn't quite right.  Triggers with this property guarantee that there will always be an ON_TIME pane, regardless of whether there are elements in that pane

Reply:
Thanks for the clarification. Modified based on your suggestion.